### PR TITLE
Refactor deployment scripts and lint

### PR DESCRIPTION
**Pull request by edmundedgar**

### CODE REVIEW

File: src/deployment/7_claimChainInfo.js [hunk 1]
```diff
@@ -1,105 +1,78 @@
...
+    const currentProvider = await common.loadProvider(deployParameters, process.env);
+    const deployer = await common.loadDeployer(currentProvider, deployParameters);
+
+    const l2BridgeFactory = await ethers.getContractFactory('@RealityETH/zkevm-contracts/contracts/inheritedMainContracts/PolygonZkEVMBridge.sol:PolygonZkEVMBridge', deployer);
     const l2BridgeContract = l2BridgeFactory.attach(l2BridgeAddress);
 
-    const sleep = ms => new Promise(r => setTimeout(r, ms));
+    const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
 
     function filterClaimable(_depositsArray, _verbose) {
```
Comment (josojo) [inline src/deployment/7_claimChainInfo.js:45]: Maybe its easier to read if they are declared outside of main()?

File: src/deployment/6_chainInfoUpdate.js [hunk 2]
```diff
@@ -35,9 +24,9 @@ async function main() {
     const l1GlobalChainInfoPublisher = l1GlobalChainInfoPublisherFactory.attach(l1Applications.l1GlobalChainInfoPublisher);
 
     console.log('sending chain info update with addresses', l1BridgeAddress, l2Applications.l2ChainInfo);
-    const result = await l1GlobalChainInfoPublisher.updateL2ChainInfo(l1BridgeAddress, l2Applications.l2ChainInfo, ethers.constants.AddressZero, ethers.constants.AddressZero);
+    const zero = ethers.constants.AddressZero;
```
Comment (josojo) [inline src/deployment/6_chainInfoUpdate.js:27]: I would not have introduced the variable, but up to you

File: src/deployment/5_deployL2Applications.js [hunk 3]
```diff
@@ -28,227 +25,84 @@ async function main() {
...
-
-        // save an ongoing deployment
-        ongoingDeployment.adjudicationFramework = adjudicationFrameworkContract.address;
-        fs.writeFileSync(pathOngoingDeploymentJson, JSON.stringify(ongoingDeployment, null, 1));
-    } else {
-        adjudicationFrameworkContract = adjudicationFrameworkFactory.attach(ongoingDeployment.adjudicationFramework);
-        console.log('#######################\n');
-        console.log('AdjudicationFramework already deployed on: ', adjudicationFrameworkContract.address);
-    }
+    const l2ChainInfoContract = await common.loadOngoingOrDeploy(deployer, 'L2ChainInfo', 'l2ChainInfo', [l2BridgeAddress, l1GlobalChainInfoPublisher], ongoingDeployment, pathOngoingDeploymentJson);
```
Comment (josojo) [inline src/deployment/5_deployL2Applications.js:96]: much more readable. I like it!

**Review by josojo — APPROVED** [review]
So much easier to read and to maintain. Thanks

File: src/deployment/7_claimChainInfo.js (hunk 1 above)
Comment (edmundedgar) [reply src/deployment/7_claimChainInfo.js:45]: In javascript I like to keep function definitions close to where they're used with the most limited scope that'll do the job

**Pull request merged**